frontend/public/components/factory/list-page.tsx

```
@@ -296,6 +296,7 @@ export const FireMan: FC<FireManProps & { filterList?: typeof filterList }> = (p
             content: createProps.items[item],
             'data-test': `dropdown-menu-${item}`,
```

Biome (2.5.0)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)